src/runtime_src/core/edge/drm/zocl/scu.c

```diff
@@ -92,11 +92,33 @@ stat_show(struct device *dev, struct device_attribute *attr, char *buf)
 }
 static DEVICE_ATTR_RO(stat);
 
+ssize_t show_sc_pid(u32 sc_pid, char *buf)
+{
+	ssize_t sz = 0;
+	char *fmt = "%u\n";
```

Comment:
```suggestion
	char *fmt = "PID:%u\n";
```
To match all the other files would it be a good idea to have the format like this? Even if it is one element.